.gitconfig

```diff
@@ -21,7 +24,9 @@
 	ldm = diff master --name-only
 	p = push origin
 	pl = pull origin
+	pr = remote prune origin
 	rb = rebase
+	rbi = rebase -i HEAD~5
```

Comment:
Note: this assumes you only want to edit at most 5 commits back in the commit history.